flocker/node/agents/blockdevice.py

```diff
@@ -1166,10 +1213,10 @@ def attach_volume(self, blockdevice_id, host):
         documentation.
         """
         volume = self._get(blockdevice_id)
```

Comment:
Maybe throw exception if given `storage_backend_id` doesn't match `self._storage_backend_id`, since that is unsupported by this particular implementation?